pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -97,6 +97,43 @@ def __custom_date_parser(time):
     tm.assert_frame_equal(result, expected)
 
 
+@xfail_pyarrow
+def test_read_csv_with_custom_date_parser_parse_dates_false(all_parsers):
+    # GH44366
+    def __custom_date_parser(time):
+        time = time.astype(np.float_)
+        time = time.astype(np.int_)  # convert float seconds to int type
+        return pd.to_timedelta(time, unit="s")
+
+    testdata = StringIO(
+        """time e n h
+        41047.00 -98573.7297 871458.0640 389.0089
```

Comment:
Can we reduce number of columns and use small ints? This is really hard to read

Reply:
You are right, this should be better now. I was just reusing the data from `test_read_csv_with_custom_date_parser` (which could also be simplified at a later stage)